src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  margin-right: -20px;
```

Comment:
it's not good practice to set a negative margin (especially if there is another, better way to solve the problem).

Denys suggested better solution - apply it

Reply:
ok but Denys changes do not pass the tests